### ISSUE TITLE
Why I get nothing with the training result of coco128

### ISSUE THREAD

**Issue by GoAfteraDeer** [ultralytics/yolov5#4415]
After I download the code, I didn't change any configuration in the code.And I run pip install -r requirement.txt.
I run the detect.py and it works well.
Then, I downloaded coco128 and run the train.py, it works well but the result is amazing.
The loss is nan and the mAP is 0.
![image](https://user-images.githubusercontent.com/72251735/129446523-104a5528-4d1c-4ff6-99e7-7536d86a3c91.png)

I run the detect.py with the training result--best.pt and my pc's camera, but nothing be detected.
I change the source to coco128 dataset and run detect.py, the result is as smae as before, nothing be detected.
I use gtx1650,cuda11.2 and torch1.9.0 The system is win10.
This is my configuration when running train.py

![image](https://user-images.githubusercontent.com/72251735/129446865-c418f267-faa9-4bc0-bf7c-724b71bf71b8.png)
![image](https://user-images.githubusercontent.com/72251735/129446873-8b9f87d5-8b6d-4d06-b70a-c51f599f424b.png)

I am puzzle by the result. Is there something wrong with my operation?

**Comment by glenn-jocher** [comment]
@GoAfteraDeer see the Colab notebook for an example COCO128 training:
https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb

**Comment by matias-kovero** [comment]
> I run the detect.py with the training result--best.pt and my pc's camera, but nothing be detected.
> I change the source to coco128 dataset and run detect.py, the result is as smae as before, nothing be detected.

> I use gtx1650,cuda11.2 and torch1.9.0 The system is win10.
> I am puzzle by the result. Is there something wrong with my operation?

I also had issues with cuda 11.2 version of torch. I have Nvidia driver: 466.11 installed that includes CUDA 11.3.70
so I opted to install cuda 11.2 version of torch. However had the same issues as you have - `detect.py` works but `train.py` gives nan loss and 0 mAP. Also using `torch.hub` for inference won't give any inference results.

Fixed the issue by installing PyTorch cuda 10.2 version:
```
pip3 install torch==1.9.0+cu102 torchvision==0.10.0+cu102 torchaudio===0.9.0 -f 
https://download.pytorch.org/whl/torch_stable.html
```
